src/Chapter3.hs

```diff
@@ -971,6 +1115,26 @@ implement the following functions:
 🕯 HINT: to implement this task, derive some standard typeclasses
 -}
 
+data WeekDay
+  = Monday
+  | Tuesday
+  | Wednesday
+  | Thursday
+  | Friday
+  | Saturday
+  | Sunday deriving (Show, Eq, Enum)
+
+isWeekend :: WeekDay -> Bool
+isWeekend day = any ((== True) . (== day)) [Saturday,Sunday]
```

Comment:
`== True` is redundant call here because `== day` already returns `Bool` :slightly_smiling_face: